e2e/cluster_test.go

```diff
@@ -1030,59 +1031,6 @@ var _ = Describe("SpiceDBClusters", func() {
 							g.Expect(len(cluster.Status.Conditions)).To(BeZero())
 						}).Should(Succeed())
 					})
-
```

Comment:
I removed this test because there's a much better test that walks through multiple updates